otherlibs/unix/.depend

```diff
@@ -77,10 +77,10 @@ cstringv.o: cstringv.c ../../byterun/caml/mlvalues.h \
  ../../byterun/caml/freelist.h ../../byterun/caml/minor_gc.h \
  ../../byterun/caml/address_class.h ../../byterun/caml/osdeps.h \
  ../../byterun/caml/memory.h unixsupport.h
-dup.o: dup.c ../../byterun/caml/mlvalues.h ../../byterun/caml/config.h \
+dup2.o: dup2.c ../../byterun/caml/mlvalues.h ../../byterun/caml/config.h \
  ../../byterun/caml/m.h ../../byterun/caml/s.h ../../byterun/caml/misc.h \
```

Comment:
This appears to be an unrelated change.

Reply:
I just made `make alldepend`. Do you want me to revert that bit?